### ISSUE TITLE
Vertical Scroll on some mathjax formulas

### ISSUE THREAD

**Issue by AndreWin** [squidfunk/mkdocs-material#484]
Hello!

I use Mathjax from [here](https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.2/MathJax.js?config=TeX-MML-AM_CHTML).

On following formulae I get vertical scrollbar on the right hand:

```
$$
\begin{equation}
s_{na1} = d_{a1} \left ( \dfrac{\dfrac{\pi}{2} + 2x_1\tg{\alpha}}{z_1} + \inv{\alpha_t} - \inv{\alpha_{a1}} \right )\cos{\beta_{a1}}
\end{equation}
$$
```
![image](https://user-images.githubusercontent.com/7497378/31400104-1c38dd26-adf7-11e7-855e-1d6e9ee53030.png)

My mathjax config:

```
MathJax.Hub.Config({
    tex2jax: {
        inlineMath: [ ["$","$"], ["\\(","\\)"] ],
        displayMath: [ ["$$","$$"], ["\\[","\\]"] ],
        processEscapes: true,
    },
    TeX: {
        Macros: {
            Alpha: "\\mbox{A}",
            Beta: "\\mbox{B}",
            Epsilon: "\\mbox{E}",
            Zeta: "\\mbox{Z}",
            Eta: "\\mbox{H}",
            Iota: "\\mbox{I}",
            Kappa: "\\mbox{K}",
            Mu: "\\mbox{M}",
            Nu: "\\mbox{N}",
            Omicron: "\\mbox{O}",
            Rho: "\\mbox{P}",
            Tau: "\\mbox{T}",
            Chi: "\\mbox{X}",
            and: "\\mbox{&}",
            or: "\\lor",
            exist: "\\exists",
            empty: "\\emptyset",
            P: "\\mbox{P}",
            tan: "\\operatorname{tg}",   // tangent
            tg: "\\operatorname{tg}",    // tangent
            cot: "\\operatorname{ctg}",  // cotangent
            ctg: "\\operatorname{ctg}",  // cotangent
            csc: "\\operatorname{cosec}",     // cosecant
            cosec: "\\operatorname{cosec}",   // cosecant
            arctan: "\\operatorname{arctg}",  // arctangent
            arctg: "\\operatorname{arctg}",   // arctangent
            arccot: "\\operatorname{arcctg}",      // arc cotangent
            arcctg: "\\operatorname{arcctg}",      // arc cotangent
            arcsec: "\\operatorname{arcsec}",      // arc secant
            arccsc: "\\operatorname{arccosec}",    // arc cosecant
            arccosec: "\\operatorname{arccosec}",  // arc cosecant
            sh: "\\operatorname{sh}",     // hyperbolic sine
            ch: "\\operatorname{ch}",     // hyperbolic cosine
            th: "\\operatorname{th}",     // hyperbolic tangent
            cth: "\\operatorname{cth}",   // hyperbolic cotangent
            sinh: "\\operatorname{sh}",   // hyperbolic синус
            cosh: "\\operatorname{ch}",   // hyperbolic cosine
            tanh: "\\operatorname{th}",   // hyperbolic tangent
            coth: "\\operatorname{cth}",  // hyperbolic cotangent
            sgn: "\\operatorname{sgn}",
            mod: "\\operatorname{mod}",
            inv: "\\operatorname{inv}",
            ge: "\\geqslant",
            le: "\\leqslant",
            geq: "\\geqslant",
            leq: "\\leqslant",
            N: "\\mathbb{N}",
            R: "\\mathbb{R}",
            Q: "\\mathbb{Q}",
            Z: "\\mathbb{Z}",
            C: "\\mathbb{C}",
            H: "\\mathbb{H}",
            P: "\\mathbb{P}",
            dmtr: "\\unicode{x2300}", // diameter sign
            deg: "\\unicode{xb0}",    // degree sign
            celdeg: "\\unicode{x2103}"   // degree Celsius sign
        },
        // AutoNumbering of displayed formulas
        equationNumbers: { autoNumber: "AMS" },
        // All mathjax extensions:
		// http://docs.mathjax.org/en/latest/tex.html#tex-and-latex-extensions
        // Source code for all extensions:
		// https://github.com/mathjax/MathJax/tree/master/extensions/TeX
        extensions: [
        "color.js", // Color support in LaTeX
        "autobold.js", // support for \boldsymbol{...}
        "AMSmath.js",
        "AMSsymbols.js",
        "AMScd.js", // http://www.jmilne.org/not/Mamscd.pdf
        "bbox.js", // support for \bbox[options]{math}
        // "begingroup.js", // mainly for formulas localization
        "cancel.js" // support for strikethrough formulas
        // "HTML.js" // works by default
        // "mhchem.js" // chemical formulas
        // "uniconde.js" // works by default
        ]
    }
});
```

I asked my question on mathjax repo: https://github.com/mathjax/MathJax/issues/1853#issue-264308076 
Then I tested it on default mkdocs theme and I found that this bug is only when I use mkdocs material theme.

Please help me to fix it.

Best regards,
Andrey.

**Comment by squidfunk** [comment]
Can you please provide a link to the docs that you host? It's easier to fix then.

**Comment by AndreWin** [comment]
Yes, this bug is on http://protflex.ru/xgears/cyl_gear/calcs/geom_calc/
Problem formulas are two last numbered ones on this page. They are at the end of this page.

**Comment by squidfunk** [comment]
Fixed in #486, release will follow shortly.

**Comment by AndreWin** [comment]
Thank you very much!
Will this update avaliable via pip?

**Comment by squidfunk** [comment]
Yes, 1.10.3 is just being released, give it a few minutes.

**Comment by AndreWin** [comment]
Thanks you very much! ^_^

**Comment by squidfunk** [comment]
1.10.3 was released. I increased the padding, that solves it for this formula. I hope that it was some margin/padding issue. If you encounter the error again with another formula, please open a new issue.
